src/types.ts

```diff
@@ -182,6 +182,7 @@ export interface ExcalidrawProps {
     data: ClipboardData,
     event: ClipboardEvent | null,
   ) => Promise<boolean> | boolean;
+  renderTopRight?: (isMobile: boolean, appState: AppState) => JSX.Element;
```

Comment:
@dwelle do we need to pass `appState` in this prop? 
I think `isMobile` should be enough...
Also, should we rename this prop to `renderTopRightUI`?

Reply:
I think `appState` is useful (we need it ourselves: https://github.com/excalidraw/excalidraw/pull/3539/files#diff-7d537bef2a9bbed4a1fbc825d63163ce532bb20b8ea8dfc0ae1d1b296dcf2aecR298). Was wondering what signature to choose, but wanted to align with `renderFooter` which has `isMobile` as first arg.

> Also, should we rename this prop to renderTopRightUI?

I think leaving out the `UI` is fine. With `renderFooter` we don't mention `renderFooterUI` either.